tasks/src/image-to-image/about.md

```diff
@@ -1,3 +1,13 @@
+## Most Used Model for the Task
```

Comment:
@merveenoyan Can you please a little bit more clear about this change request? Do you only want the position of the 'Most Used Model for the Task' to be changed? :)

Reply:
Sure! So for every task page we have a section for the models but it's in `data.ts` rather than in the markdown file (See below) But I liked your submission that we can have it in about.md but at the end of the page, given other sections are more general and this section will be a bit specific.

<img width="607" alt="Ekran Resmi 2022-10-13 12 35 48" src="https://user-images.githubusercontent.com/53175384/195574857-556786ea-c54a-4490-9894-90083593d542.png">

Reply:
Ok, I have made the changes you asked for. Thanks for all the feedbacks and the kind replies. I will try to contribute more to the project in the following days. I just want to give my apologies for the mistake but I will learn better what is desired with time :)